cognite/experimental/_api/annotations_v2.py

```
@@ -26,6 +27,34 @@ def create(self, annotations: Union[AnnotationV2, List[AnnotationV2]]) -> Union[
         assert_type(annotations, "annotations", [AnnotationV2, list])
         return self._create_multiple(resource_path=self._RESOURCE_PATH + "/", items=annotations)
 
+    def suggest(self, annotations: Union[AnnotationV2, List[AnnotationV2]]) -> Union[AnnotationV2, AnnotationV2List]:
+        """Suggest annotations
+
+        Args:
+            annotations (Union[AnnotationV2, List[AnnotationV2]]): annotation(s) to suggest. They must have status set to "suggested".
+
+        Returns:
+            Union[AnnotationV2, AnnotationV2List]: suggested annotation(s)
+        """
+        assert_type(annotations, "annotations", [AnnotationV2, list])
+        # Deal with status fields in both cases: Single item and list of items
+        items = (
+            [AnnotationsV2API._sanitize_suggest_item(ann) for ann in annotations]
+            if isinstance(annotations, list)
+            else AnnotationsV2API._sanitize_suggest_item(annotations)
+        )
+        return self._create_multiple(resource_path=self._RESOURCE_PATH + "/suggest", items=items)
+
+    @staticmethod
+    def _sanitize_suggest_item(annotation: AnnotationV2) -> Dict[str, any]:
+        # Check that status is set to suggested if it is set and afterwards remove it
+        item = annotation.dump(camel_case=True) if isinstance(annotation, AnnotationV2) else deepcopy(annotation)
+        if "status" in item:
+            if item["status"] != "suggested":
+                raise ValueError("status field for Annotation suggestions must be set to 'suggested'")
+            del item["status"]
```

Whether we need to protect the user from their obvious mistakes or not is a bit up to you as the author of the PR. But my take on it is it's perfectly fine to let the request go through to the API, and let the API fail. In that way we also eliminate redundant logic here in the SDK.

---

You are right and I was debating that actually. It is completely ok to fail. 
I decided in the end to make it explicitly fail because the user is forced to set the "status" field when creating an instance of the `AnnotationV2` dataclass and if they set it to "approved" or "rejected" they would get very confused if the created annotations silently end up as "suggested"